integration/schema.graphql

```diff
@@ -8,6 +8,7 @@ type Author implements Named {
   working: Working
   birthday: Date
   books: [Book!]!
+  bookPopularities: [PopularityDetail!]!
```

Comment:
Maybe include an enum detail list that is nullable, i.e. `[PopularityDetail!]`? Maybe that will just work, not sure...